src/query/algorithms/entityresolution/BloomFilter.hpp

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
Need to move this file from `.hpp` to `.h` file. We had discussed about this multiple times before during offline discussions.

Reply:
Changed to .h extension